wls-auth-service/src/main/resources/db/migrations/h2/V4_9__insertWahlvorbereitungPermissions.sql

```diff
@@ -0,0 +1,177 @@
+INSERT INTO Secauthorities_Secpermissions
+VALUES ('00000000-0000-0000-0001-000000000001', '00000000-0000-0000-0009-000000000021');
+
+INSERT INTO Permission
+VALUES ('00000000-0000-0000-0009-000000000022', 'Wahlvorbereitung_WRITE_UrnenwahlSchliessungsuhrzeit');
+INSERT INTO Secauthorities_Secpermissions
+VALUES ('00000000-0000-0000-0001-000000000001', '00000000-0000-0000-0009-000000000022');
+
+INSERT INTO Permission
+VALUES ('00000000-0000-0000-0009-000000000023', 'Wahlvorbereitung_DELETE_UrnenwahlSchliessungsuhrzeit');
+INSERT INTO Secauthorities_Secpermissions
+VALUES ('00000000-0000-0000-0001-000000000001', '00000000-0000-0000-0009-000000000023');
+
+-- Urnenwahlvorbereitung
+INSERT INTO Permission
+VALUES ('00000000-0000-0000-0009-000000000024', 'Wahlvorbereitung_BUSINESSACTION_GetUrnenwahlVorbereitung');
+INSERT INTO Secauthorities_Secpermissions
+VALUES ('00000000-0000-0000-0001-000000000001', '00000000-0000-0000-0009-000000000024');
+
+INSERT INTO Permission
+VALUES ('00000000-0000-0000-0009-000000000025', 'Wahlvorbereitung_BUSINESSACTION_PostUrnenwahlVorbereitung');
+INSERT INTO Secauthorities_Secpermissions
+VALUES ('00000000-0000-0000-0001-000000000001', '00000000-0000-0000-0009-000000000025');
+
+INSERT INTO Permission
+VALUES ('00000000-0000-0000-0009-000000000026', 'Wahlvorbereitung_WRITE_UrnenwahlVorbereitung');
+INSERT INTO Secauthorities_Secpermissions
+VALUES ('00000000-0000-0000-0001-000000000001', '00000000-0000-0000-0009-000000000026');
+
+INSERT INTO Permission
+VALUES ('00000000-0000-0000-0009-000000000027', 'Wahlvorbereitung_DELETE_UrnenwahlVorbereitung');
+INSERT INTO Secauthorities_Secpermissions
+VALUES ('00000000-0000-0000-0001-000000000001', '00000000-0000-0000-0009-000000000027');
+
+-- Waehlerverzeichnis
+INSERT INTO Permission
+VALUES ('00000000-0000-0000-0009-000000000028', 'Wahlvorbereitung_BUSINESSACTION_GetWaehlerverzeichnis');
+INSERT INTO Secauthorities_Secpermissions
+VALUES ('00000000-0000-0000-0001-000000000001', '00000000-0000-0000-0009-000000000028');
+
+INSERT INTO Permission
+VALUES ('00000000-0000-0000-0009-000000000029', 'Wahlvorbereitung_BUSINESSACTION_PostWaehlerverzeichnis');
+INSERT INTO Secauthorities_Secpermissions
+VALUES ('00000000-0000-0000-0001-000000000001', '00000000-0000-0000-0009-000000000029');
+
+INSERT INTO Permission
+VALUES ('00000000-0000-0000-0009-000000000030', 'Wahlvorbereitung_WRITE_Waehlerverzeichnis');
+INSERT INTO Secauthorities_Secpermissions
+VALUES ('00000000-0000-0000-0001-000000000001', '00000000-0000-0000-0009-000000000030');
+
+INSERT INTO Permission
+VALUES ('00000000-0000-0000-0009-000000000031', 'Wahlvorbereitung_DELETE_Waehlerverzeichnis');
+INSERT INTO Secauthorities_Secpermissions
+VALUES ('00000000-0000-0000-0001-000000000001', '00000000-0000-0000-0009-000000000031');
+
+-- Urnenwahlvorbereitung
+INSERT INTO Permission
+VALUES ('00000000-0000-0000-0009-000000000032', 'Briefwahl_READ_Wahlbriefdaten');
```

Comment:
ist briefwahl richtig? und wenn ja, muss es dann nicht in das `insertBriefwahlPermissions`-File? wenn es hier richtig ist, muss es in `Wahlvorbereitung` umbenannt werden
```suggestion
VALUES ('00000000-0000-0000-0009-000000000032', 'Wahlvorbereitung_READ_Wahlbriefdaten');
```

Reply:
da ist mal etwas richtig durcheinander geraten. Habe es zu den Briefwahlpermissions geschoben

fixed mit 18b00bfe19d46d7ca787c50973ea12210b69716f